3rdparty/bfloat16/bfloat16.cc

```diff
@@ -1,84 +0,0 @@
-/*
-    Copyright (c) 2019 by Contributors
-   \file tvm/src/codegen/custom_datatypes/mybfloat16.cc
```

Comment:
It would still be great to have a library independent examples

Reply:
@hypercubestart is working on reviving the old "actually-float32-under-the-hood" type that I had originally included.